stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1147,18 +1147,19 @@ void DefaultActorImpl::enqueue(Job *job, JobPriority priority) {
 #if SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
       if (oldState.getMaxPriority() != newState.getMaxPriority()) {
         if (newState.isRunning()) {
-          // Actor is running on a thread, escalate the thread running it
           SWIFT_TASK_DEBUG_LOG("[Override] Escalating actor %p which is running on %#x to %#x priority", this, newState.currentDrainer(), priority);
           dispatch_lock_t *lockAddr = this->drainLockAddr();
           swift_dispatch_lock_override_start_with_debounce(lockAddr, newState.currentDrainer(),
                          (qos_class_t) priority);
         } else {
-          // TODO (rokhinip): Actor is scheduled - we need to schedule a
-          // stealer at the higher priority
-          //
-          // TODO (rokhinip): Add a signpost to flag that this is a potential
-          // priority inversion
-          SWIFT_TASK_DEBUG_LOG("[Override] Escalating actor %p which is enqueued", this);
+          // We are scheduling a stealer for an actor due to priority override.
+          // This extra processing job has a reference on the actor. See
+          // ownership rule (2).
+          SWIFT_TASK_DEBUG_LOG(
+              "[Override] Scheduling a stealer for actor %p at %#x priority",
+              this, newState.getMaxPriority());
+          swift_retain(this);
+          scheduleActorProcessJob(newState.getMaxPriority());
```

Comment:
Please change the stale comments in scheduleActorProcessJob since there can now be multiple processing jobs for an actor.